python/services/evaluationservice/dmod/evaluationservice/service/application_values.py

```diff
@@ -49,7 +49,10 @@ def get_redis_password() -> typing.Optional[str]:
     Returns:
         The optional password to the core redis service
     """
-    password_filename = os.environ.get("REDIS_PASSWORD_FILE", "/run/secrets/myredis_pass")
+    if password_path_variable is None:
+        password_path_variable = "REDIS_PASSWORD_FILE"
```

Comment:
This is a protective approach - `get_redis_password(None)` or `get_redis_password(password_path_variable=None, password_variable_name="example")` will throw an error if the pattern of the default lying in the signature is used instead of the "Expect `None` and provide a default" approach. This helps prevent issues when another function calls `get_redis_password` with a variable that never got a valid value. `get_redis_password(might_be_null)` will now work.

I can migrate the defaults to the signature if you have a strong preference, though.